okhttp-tests/src/test/java/okhttp3/HttpUrlTest.java

```diff
@@ -1348,6 +1355,29 @@
     }
   }
 
+  @Test public void getCheckedMethodAccess() throws Exception {
```

Comment:
I don't think this is a good precedent for checking we don't break API.  We can check this in other ways, but we could have hundreds of these tests and they have low value.

Reply:
@yschimke test deleted